src/basho_bench_duration.erl

```diff
@@ -96,20 +96,32 @@ terminate(Reason, #state{duration=DurationMins}) ->
         WSup ->
             WRef = erlang:monitor(process, WSup),
             supervisor:terminate_child(basho_bench_run_sup, basho_bench_worker_sup),
+            supervisor:terminate_child(basho_bench_run_sup, basho_bench_stats),
```

Comment:
I remembered it was helpful in the post-hook stats from my earlier notes but I can remove it for now.